### DISCUSSION TITLE
ILS landing nosedives at ~40 feet

### DISCUSSION THREAD

**Discussion by Skk-tj** [flybywiresim/aircraft#3040] [Q&A] [answered]
I am using the latest version. I was trying to land on KSEA via ILS approach, with AP1 and APPR on and left untouched the entire time. When GPWS announces 40 or 50 feet the plane suddenly just nosedives to the ground. Is this to be expected?

**Comment by Skk-tj** [comment, marked as answer]
#2997 
Turns out I'll have to disable autopilot at ~100ft

**Reply by metronorth** [reply]
What you're encountering is a simulator approximation of 'flare mode'. I think it's maybe not OK to post YouTube links here(?) but if ya do a YT search for A320 flare mode you'll see the 10+ minute video I would like to recommend. Flare mode induces a nose-down pitch roughly 30 feet AGL or so off the top of my head. That's meant to induce a flare on part of the pilot flying, from my understanding. Why that's required to induce an actual flare, I'm not sure. Probably has to do with the kind of strange, but sophisticated, ways that Airbus has developed their flight control laws. The solution you came up with on your own seems to work fine for myself, as well. But what about "auto-land" as at least many other aircraft would call it? They don't have to de-clutch or disengage autopilot systems entirely afaik. Much of this is probably in various manuals such as the Performance Training Manual (different from the FCOM and FTCM and quite handy). [https://www.cockpitseeker.com/wp-content/uploads/goodies/ac/a320/pdf/Print_Only/PTM%20with%20airbus%20doc/pdf/U0S2SP0-L.pdf](https://www.cockpitseeker.com/wp-content/uploads/goodies/ac/a320/pdf/Print_Only/PTM%20with%20airbus%20doc/pdf/U0S2SP0-L.pdf) Good luck!

Addendum:

In fact, I'll just quote the synopsis from the AirbusDriver website on Flare Mode as it might overall be more helpful and concise: 

- Transition to flare mode occurs at **50' RA** during landing.
- System _**memorizes pitch attitude at 50' and begins to progressively reduce pitch**_, forcing pilot to flare the aircraft
- In the event of a go-around, transition to flight mode occurs again at 50' RA.

**Reply by Skk-tj** [reply]
Thanks for the information. I think I found the video you mentioned. The situation I encountered was a little bit more dramatic than what the video depicted: the aircraft literally just slams to the runway in less than a second.

**Reply by robinottawa** [reply]
Me too.
